### DISCUSSION TITLE
New firmare release for V2 (v2.40?)

### DISCUSSION THREAD

**Discussion by WilfredHoogerbrugge** [SmartEVSE/SmartEVSE-2#59] [Q&A] [answered]
The latest firmware version available (in SmartEVSE2.X/dist/default/production) is v2.30 which is almost 3 years old.
There has been plenty of development afterwards and it seems to contain a fix ([Compare start current value with sum of available current](https://github.com/SmartEVSE/SmartEVSE-2/commit/66f52e6327f50f96bcd70df9e903cb975818917b)) for a solar charging problem I have.

I know (and understand) the main focus is now on the V3 hardware but it would be nice to have a new 'official' release for the V2 hardware containing all the fixes that are already made. I would happily volunteer to test it...

**Comment by bobosch** [comment, marked as answer]
Hi! Here is a compiled binary from the latest version. [SmartEVSE2.X.production.hex.zip](https://github.com/user-attachments/files/16830558/SmartEVSE2.X.production.hex.zip)

**Reply by WilfredHoogerbrugge** [reply]
Thanks for this supplying this 'snapshot' version.

I installed it on my device and (after fixing the configuration which was cleared during installation) all seems to work fine.
The problem I had with PV production detection due to PV production on 3 phases is fixed with this release.
The car is charging in solar mode now and this seems to work fine. If I notice problems I will report back.

BTW: The version reported by this firmware is still v2.30 which is a little confusing, I would suggest to give these kind of snapshot release a version which makes it clear it is not an official release.

**Reply by bobosch** [reply]
Thanks for testing :-)
